### ISSUE TITLE
Login fails in Ubuntu 24.04

### ISSUE THREAD

**Issue by jeffguy** [regolith-linux/regolith-desktop#1010]
- [X] I've [run a search on the existing bug reports](https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests) to make sure I'm not filing a duplicate (you can use the search box at the top left corner)
- [X] I've read [the Regolith documentation](https://regolith-linux.org/docs/) and didn't find any relevant topic
- [X] I tried asking on Matrix and Reddit (especially for configuration related issues)

**Describe the bug**
 On a fresh Ubuntu 24.04, I installed Regolith3.1. On reboot, when I selected Regolith and logged in, the screen went black for 5-10 seconds, then returned to the login.

When I tried the instructions in the 3.2 release notes, it didn't update the /etc/regolith/version file. Still, I tried rebooting into regolith and had the same symptoms.

Finally, I used dpkg --get-selections|grep regolith and dpkg --get-selections|grep i3 to remove everything that looked regolith- or i3-related. Then I reinstalled regolith-3.2. /etc/regolith has two times: i3 and picom. /etc/regolith/version is conspicuously absent. Also, when I reboot into regolith, I see the same symptoms.

**To Reproduce**
Install fresh Ubuntu 24.04.
Install Regolith 3.1.
Reboot.
Attempt login with Regolith.
Observe failure as described above.
Login with Gnome.
Adjust /etc/apt.d/sources file to point to testing (instead of regolith 3.1).
apt update and upgrade.
Reboot
Attempt login with Regolith.
Observe same failure.
Observe /etc/regolith/version is still 3.1
Use dpg --get-selections|grep regolith (and i3) to remove everything regolith-related or i3-related.
Delete /etc/regolith/version if it's still there.
Install regolith 3.2.
Reboot.
Attempt login with Regolith.
Observe same failure.
Observe /etc/regolith/version doesn't exist.

**Expected behavior**
I expect that when I install Regolith 3.2 on Ubuntu 24.04, it will create an /etc/regolith/version file with the correct version (testing or 3.2 are both acceptable), and when I reboot, I will enter Regolith Desktop (visible status bar at bottom, ability to launch terminal, etc.)

**Screenshots**
N/A. It's just a black screen for 5-10 seconds, then a return to the login screen.

**Configuration file(s)**
/etc/apt/sources.list.d/regolith.list
deb [arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] https://regolith-desktop.org/testing-ubuntu-mantic-amd64 mantic main

/etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=24.04
DISTRIB_CODENAME=noble
DISTRIB_DESCRIPTION="Ubuntu 24.04 LTS"

I didn't mess with any config files. They're the defaults for a fresh install of Regolith 3.1 and Regolith 3.2.

**Installation Details**
 - Regolith Install Type: PPA
 - Regolith Version: 3.2
 - PPA url: Is this just the line in the /etc/apt/sources file? https://regolith-desktop.org/testing-ubuntu-mantic-amd64
 - Host OS (for PPA): Ubuntu 24.04

**Additional context**
Thanks for your hard work. I've been using Regolith on other computers for about a year, and it makes my computing experience so much smoother.

**Comment by jeffguy** [comment]
If the path I followed to install Regolith 3.2 was the wrong path, maybe I misunderstood the documentation. Here's the trail I used:
https://regolith-desktop.com/docs/using-regolith/install/ (therefore installed 3.1, hoping for the best)
https://regolith-desktop.com/docs/reference/Releases/regolith-3.2-release-notes/ (therefore updated /etc/apt/sources and checked /etc/regolith/version file)

**Comment by kgilmer** [comment]
Thanks again for the details @jeffguy .  I believe your issues will be resolved if you update the apt line as specified in the [3.2 release notes.](https://regolith-desktop.com/docs/reference/Releases/regolith-3.2-release-notes/)  In particular, `/etc/apt/sources.list.d/regolith.list` should be the following for Ubuntu Noble 24.04:

```
echo deb "[arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] \
https://regolith-desktop.org/testing-ubuntu-noble-amd64 noble main" | \
sudo tee /etc/apt/sources.list.d/regolith.list
```

The info you posted above contains a URL for the previous Ubuntu version.

**Comment by jeffguy** [comment]
Great catch! I fixed that. apt update;apt upgrade didn't fix it. Then I used
dpkg --get-selections|egrep 'i3|regolith'|tr '\t' ' '|cut -d' ' -f1|grep -v libpci3|grep -v libxcb-dri3-0
to get rid of a bunch of packages and then
sudo apt install regolith-desktop regolith-session-flashback regolith-look-lascaille
and after a reboot it worked perfectly!

Thank you so much for your hard work on Regolith and for helping me with this issue in particular!
